docs/development/labs.md

```diff
@@ -0,0 +1,508 @@
+---
+title: Creating Labs preview features
+description: Learn how to add preview features to Home Assistant Labs
+---
+
+Home Assistant Labs provides a standardized way to ship preview features that users can opt into before they become standard. This guide explains how to add a new preview feature to Labs.
+
+## What are Labs preview features?
+
+Labs preview features are critical bug free features that are being refined through real-world usage and feedback before becoming standard in Home Assistant. While fully functional, their feature set may still be extended or refined based on user feedback. They differ from beta testing, which evaluates release stability.
+
+## When to use Labs
+
+Labs is appropriate for:
+
+- Major UI changes or redesigns that benefit from user feedback
+- New subsystems or significant architectural changes
+- Features that need real-world testing to refine the user experience
+- Functionality where user feedback will shape the final design
```

Comment:
tbf, the first, third and fourth kinda say the same thing, features that need testing and feedback